mods/content/beekeeping/hives/hive_swarm.dm

```diff
@@ -0,0 +1,374 @@
+/obj/effect/insect_swarm
+	anchored          = TRUE
+	is_spawnable_type = FALSE
+	icon_state        = "0"
+	gender            = NEUTER
+	default_pixel_z   = 8
+	layer             = ABOVE_HUMAN_LAYER
+	pass_flags        = PASS_FLAG_TABLE
+	movement_handlers = list(/datum/movement_handler/delay/insect_swarm)
```

Comment:
```dm
/datum/movement_handler/delay/New(var/host, var/delay)
	..()
	src.delay = max(1, delay)
```
it's gonna get its delay set wrong bc of this inheritance, maybe `/delay` needs an isnull to avoid setting it if we pass null. that or just do `list(/datum/movement_handler/delay = 1 SECOND)`